recipes/python-ray/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "ray" %}
+{% set version = "0.6.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ray-project/{{ name }}/archive/ray-{{ version }}.tar.gz
+  sha256: e606d3ead8573b2f34a2bbc339e2dca9f1125abcf89d8f430152ff21a0362568
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install python/ --no-deps -vv"
```

This will fail on Windows but it's `noarch` so it doesn't matter:
```suggestion
  script: "cd python && {{ PYTHON }} -m pip install . --no-deps -vv"
```